### PR TITLE
Updating the copyright date

### PR DESCRIPTION
**Pull request by slash1andy**
Just a quick update I noticed when doing a theming workshop at our local meetup.

#### Changes proposed in this Pull Request:

Date updated

#### Related issue(s):

### CODE REVIEW

**Comment by grappler** [comment]
Thank you @slash1andy for the PR. The date can also be updated in the `sass/style.sccs`, `reame.txt` and the `langauges/_s.pot` file,

**Comment by slash1andy** [comment]
@grappler done! Thanks for the feedback on such a simple PR.

File: readme.txt [hunk 1]
```diff
@@ -34,5 +34,5 @@ _s includes support for Infinite Scroll in Jetpack.
 
 == Credits ==
 
-* Based on Underscores https://underscores.me/, (C) 2012-2017 Automattic, Inc., [GPLv2 or later](https://www.gnu.org/licenses/gpl-2.0.html)
+* Based on Underscores https://underscores.me/, (C) 2012-2018 Automattic, Inc., [GPLv2 or later](https://www.gnu.org/licenses/gpl-2.0.html)
 * normalize.css https://necolas.github.io/normalize.css/, (C) 2012-2016 Nicolas Gallagher and Jonathan Neal, [MIT](https://opensource.org/licenses/MIT)
```
[the remark this answers is not in the document]
Comment (slash1andy) [reply readme.txt]: I looked at the Normalize Repo, but there was no updated copyright date that I could find.

[the remark this answers is not in the document]
Comment (Ismail-elkorchi) [reply readme.txt]: _s uses Normalize v8.0.0, which was published in 2018. That's what pushed me to think that its copyright needs to be updated too.

[the remark this answers is not in the document]
Comment (slash1andy) [reply readme.txt]: I agree, have updated. Thanks.

**Comment by grappler** [comment]
@slash1andy The changes look good. Do you think you will be able to squash all of the commit into one?

**Comment by slash1andy** [comment]
Ok, I'm not going to lie. I tried to squash it, but only succeeded in muddying it further. Advice on how to remedy this?

**Comment by Ismail-elkorchi** [comment]
Closing as this was fixed in https://github.com/Automattic/_s/pull/1389 and https://github.com/Automattic/_s/pull/1390.

**Pull request closed without merging**